python/packages/autogen-core/src/autogen_core/_cache_store.py

```diff
@@ -0,0 +1,34 @@
+from typing import Any, Optional, Protocol
+
+
+class CacheStore(Protocol):
+    """
+    This protocol defines the basic interface for store/cache operations.
+
+    Allows duck-typing with any object that implements the get and set methods,
+    such as redis or diskcache interfaces.
+    """
+
+    def get(self, key: str, default: Optional[Any] = None) -> Optional[Any]:
```

Comment:
What happened to the types here? The duck-typing scenario is still valid if an API expects Any as the type. I really don't think we should be using an untyped interface